src/org/skypro/skyshop/basket/ProductBasket.java

```diff
@@ -0,0 +1,62 @@
+package org.skypro.skyshop.basket;
+
+import org.skypro.skyshop.product.Product;
+
+public class ProductBasket {
+
+    private final Product[] basket;
+    private int productCounter;
+
+    public ProductBasket() {
+        basket = new Product[5];
+        productCounter = 0;
+    }
+
+    public void addProduct(Product product) {
+        if (productCounter < 5) {
```

Comment:
почему не 10? надо использовать длину массива, а не число с потолка
```suggestion
        if (productCounter < basket.length) {
```